docs/api/version-history.md

```diff
@@ -27,6 +27,8 @@ keywords: "API, Docker, rcli, REST, documentation"
   `--sysctl` settings for `eth0` will be migrated to `DriverOpts` when possible. 
   This automatic migration will be removed for API versions 1.47 and greater.
 
+* `GET /containers/json` now returns the annotations of containers.
```

Comment:
```suggestion
* `GET /containers/json` now returns the annotations of containers.
```

nit: Extra empty line 🙈

Reply:
Heh, yeah, I noticed that one, but knew there would be other PRs incoming touching this file, so we can fix in those 🙈